### ISSUE TITLE
[🚫 Bug] Downloading chrome driver does currently not work

### ISSUE THREAD

**Issue by rH4rtinger** [redhat-developer/vscode-extension-tester#1340]
### Describe the bug

Currently, when running any command that downloads the chrome driver like `npx extest get-chromedriver`, it will fail, because the given URL returns a 404.

The url https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/122.0.6261.128/win64/chromedriver-win64.zip does currently not exists.

However, if you were looking at https://github.com/GoogleChromeLabs/chrome-for-testing#json-api-endpoints and searching https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json for the driver with the number, you would find a driver and a download link.
Maybe you should use this json for checking the drivers, because it has much more information than https://googlechromelabs.github.io/chrome-for-testing/ . This site only has drivers beginning with version 125.x

### Steps to reproduce

```Shell
1. npx extest get-chromedriver
2. see console output
```

### Logs

```Shell
> npx extest get-chromedriver

Downloading ChromeDriver 122.0.6261.128 from: https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/122.0.6261.128/win64/chromedriver-win64.zip
progress: 0/1449 (0%)
HTTPError: Response code 404 (Not Found)
    at Request._onResponseBase (file:///C:/dev/myProject/node_modules/vscode-extension-tester/node_modules/got/dist/source/core/index.js:706:31)
    at Request._onResponse (file:///C:/dev/myProject/node_modules/vscode-extension-tester/node_modules/got/dist/source/core/index.js:768:24)
    at ClientRequest.<anonymous> (file:///C:/edev/myProject/node_modules/vscode-extension-tester/node_modules/got/dist/source/core/index.js:786:23)
    at Object.onceWrapper (node:events:633:26)
    at ClientRequest.emit (node:events:530:35)
    at HTTPParser.parserOnIncomingClient (node:_http_client:698:27)
    at HTTPParser.parserOnHeadersComplete (node:_http_common:119:17)
    at TLSSocket.socketOnData (node:_http_client:540:22)
    at TLSSocket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:559:12)
```

### Operating System

Windows 10

### Visual Studio Code

1.90

### vscode-extension-tester

8.2.0

### NodeJS

18.20.3

### npm

10.8.1

**Comment by gennadylisiansky** [comment]
Same for us.

**Comment by swathisuja** [comment]
same error for linux os

**Comment by AashnaP22** [comment]
Same for mac

**Comment by MaslovMaximV** [comment]
Same for linux (ubuntu- Ubuntu 22.04.4 LTS)
```
Downloading ChromeDriver 122.0.6261.128 from: https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/122.0.6261.128/linux64/chromedriver-linux64.zip
progress: 0/1449 (0%)
HTTPError: Response code 404 (Not Found)
    at Request._onResponseBase (file:///home/maxim/project/autotest/amplicode-vscode-extension/node_modules/got/dist/source/core/index.js:706:31)
    at Request._onResponse (file:///home/maxim/project/autotest/amplicode-vscode-extension/node_modules/got/dist/source/core/index.js:768:24)
    at ClientRequest.<anonymous> (file:///home/maxim/project/autotest/amplicode-vscode-extension/node_modules/got/dist/source/core/index.js:786:23)
    at Object.onceWrapper (node:events:628:26)
    at ClientRequest.emit (node:events:525:35)
    at HTTPParser.parserOnIncomingClient (node:_http_client:693:27)
    at HTTPParser.parserOnHeadersComplete (node:_http_common:128:17)
    at TLSSocket.socketOnData (node:_http_client:534:22)
    at TLSSocket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:315:12)
```

**Comment by djelinek** [comment]
Hello, thank you for reporting that! I am already working on fix

**Comment by djelinek** [comment]
Related to #1310

**Comment by swathisuja** [comment]
How to get the bug fix? Upgrading the Extester does not resolve the issue.

**Comment by apupier** [comment]
you need to wait a new release. i tis fixed on the main branch only for now

**Comment by AashnaP22** [comment]
By when should we expect the new release?

**Comment by djelinek** [comment]
Hi everyone, I will release a new version in a few hours 🙂 Thanks for your patience.

**Comment by djelinek** [comment]
Published [v8.3.0](https://github.com/redhat-developer/vscode-extension-tester/releases/tag/v8.3.0)
